packages/trader/src/Stores/Modules/Trading/Helpers/contract-type.ts

```diff
@@ -84,6 +85,7 @@ export const ContractType = (() => {
     let available_contract_types: ReturnType<typeof getContractTypesConfig> = {};
     let available_categories: TAvailableCategories = {};
     let contract_types: ReturnType<typeof getContractTypesConfig>;
+    let non_available_categories: TAvailableCategories = {};
```

Comment:
```suggestion
    let non_available_categories: TNonAvailableCategories = {};
```

Reply:
@henry-deriv this thing it has the same type as TAvailableCategories :) I've renamed both of them to `TCategories`👍🏻